sonar-plugin-api/src/main/java/org/sonar/api/issue/action/Actions.java

```diff
@@ -29,6 +30,7 @@
  */
 @Deprecated
 @ServerSide
+@ComputeEngineSide
 public class Actions {
```

Comment:
Initially, I had added it because it's in the API.

But, you're right, it does not make sense for any Component loaded on the CE side to rely on it, so I'll remove it.